doc/topics/releases/version_numbers.rst

```diff
@@ -81,21 +85,19 @@ Influence of the ``git checkout`` argument on ``git describe``:
   +------------+----------------------------+-----------------------------------------------+
   | Checkout   | Describe                   |               Comment                         |
   +============+============================+===============================================+
-  | v2016.11   | v2016.11                   | (tag is fixed point in time)                  |
-  +------------+----------------------------+-----------------------------------------------+
-  | 2016.11    | v2016.11.1-220-g9a1550d    | Commit of most recent tag in 2016.11          |
+  | v3000      | 3000                       | (tag is fixed point in time)                  |
   +------------+----------------------------+-----------------------------------------------+
-  | v2016.11.1 | 2016.11.1                  | (tag is fixed point in time)                  |
+  | v3000.1    | 3000.1                     | (tag is fixed point in time)                  |
   +------------+----------------------------+-----------------------------------------------+
-  | develop    | v2016.11.1-1741-g10d5dec   | Commit of most recent tag in develop          |
+  | master     | v3000.1-9-g10d5dec         | Commit of most recent tag in master           |
   +------------+----------------------------+-----------------------------------------------+
 
 Some details of v2016.11.1-220-g9a1550d (from ``git describe`` after ``git checkout 2016.11``):
```

Comment:
might change this to `git checkout master`. Granted, all of the hash/commits since will be different, but the fundamentals are still correct.